lib/services/BiometricAuthService.dart

```diff
@@ -17,28 +20,54 @@ Future<bool> authenticateFingerprint(int notificationId) async {
     print(e);
     return false;
   }
-  // At the last notification push, reset all notifications with new randomised values
-  if (notificationId == 2) {
-    int hours;
-    int minutes;
-
-    // This is the morning notification - Between 9:00 and 12:00
-    hours = 9 + Random().nextInt(11 - 9);
-    minutes = 0 + Random().nextInt(30 - 0);
-    resetDailyNotificationTime(
-        0, hours, minutes, 00, 'doBiometrics-$notificationId');
 
-    // This is the afternoon notification - Between 14:00 and 16:00
-    hours = 14 + Random().nextInt(16 - 14);
-    minutes = 0 + Random().nextInt(30 - 0);
-    resetDailyNotificationTime(
-        1, hours, minutes, 00, 'doBiometrics-$notificationId');
+  return authenticated;
+}
 
-    // This is the night notification - Between 18:00 and 20:00
-    hours = 18 + Random().nextInt(20 - 18);
-    minutes = 0 + Random().nextInt(30 - 0);
-    resetDailyNotificationTime(
-        2, hours, minutes, 00, 'doBiometrics-$notificationId');
+/// Makes a re-random of sthe next daily notification using the following logic:
+///
+/// If [id] == `morningDaily` -> Re-random `LocalNotificationIds.middayDaily`
+///
+/// If [id] == `middayDaily` -> Re-random `LocalNotificationIds.afternoonDaily`
+///
+/// If [id] == `afternoonDaily` -> Re-random `LocalNotificationIds.morningDaily`
+void reRandomDailyNotifications(notificationId) {
```

Comment:
I'll change the method name to something less cryptic like `reOrderDailyNotifications`